app/admin/universities/page.tsx

```
@@ -0,0 +1,512 @@
+              <Button onClick={handleUpdate}>Update University</Button>
+            </DialogFooter>
+          </DialogContent>
+        </Dialog>
+
+        {/* Delete Dialog */}
+        <Dialog open={deleteDialogOpen} onOpenChange={setDeleteDialogOpen}>
+          <DialogContent>
+            <DialogHeader>
+              <DialogTitle>Delete University</DialogTitle>
+              <DialogDescription>
+                Are you sure you want to delete "{selectedUniversity?.name}"? This action cannot be undone.
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)